admin/views/sign-in.ejs

```diff
@@ -20,7 +20,7 @@
                        aria-required="true"
                        aria-label="Enter your user name."
                        spellcheck="false"
-                       autocorrect="off">
+                       autocomplete="off">
```

Comment:
`autocorrect` is, I think, needed for iOS.

Reply:
@jon-shipley same as above.